app/CocoaPods/CPPodfileEditorViewController.swift

```diff
@@ -44,6 +54,13 @@ class CPPodfileEditorViewController: NSViewController, NSTextViewDelegate {
     syntaxChecker.textDidChange(NSNotification(name: "", object: nil))
   }
 
+  func completions() -> [AnyObject]! {
```

Comment:
Unfortunately, this is how it's declared in `SMLAutoCompleteDelegate`. If I change the return type in the implementation, the build errors for non protocol conformance.